x-pack/plugins/ml/server/routes/job_service.ts

```diff
@@ -272,6 +272,41 @@ export function jobServiceRoutes({ router, routeGuard }: RouteInitialization) {
     })
   );
 
+  /**
+   * @apiGroup JobService
+   *
+   * @api {post} /api/ml/jobs/jobs_for_export Get job
+   * @apiName CreateFullJobsList
```

Comment:
This has been copied from another endpoint and should be changed to match the route, such as `GetJobsListForExport` or `GetJobsListForCloning`

Reply:
Updated here https://github.com/elastic/kibana/pull/88898/commits/37e73ae597cc4acd9db6db59a17ec893799d240d